src/index.html

```diff
@@ -11,7 +11,168 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="POST"
+    onsubmit="onSubmit()">
+      <fieldset>
+        <legend>Personal information</legend>
+        <div class="field">
+          <label for="surname">
+              Surname:
+            <input type="text" id="surname"
+            autocomplete="off"
+            required
+            >
+          </label>
+        </div>
+
+        <div class="field">
+          <label for="name" class="innerform">
+              Name:
+            <input type="text" id="name"
+            autocomplete="off"
+            required
+            >
+          </label>
+        </div>
+
+        <div class="field">
+          <label for="age" class="innerform">
+              How old are You?
+            <input type="number" id="age"
+            value="12"
+            max="100"
+            min="0"
+            >
+          </label>
+        </div>
+
+        <div class="field">
+          <label for="fulldate" class="innerform">
+              Full date of birth:
+            <input type="date" id="fulldate">
+          </label>
+        </div>
+
+        <div class="field">
+          <label for="agree" class="innerform">
+              I accept the term of the agreement
+            <input type="checkbox" id="agree">
+          </label>
+        </div>
+
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend>Registration</legend>
+        <div class="field">
+          <label for="email">
+            E-mail: <input type="email" id="email"
+            placeholder="email@example.com"
+            required
+            >
```

Comment:
Also, start tag from new line.
```suggestion
            E-mail:
            <input
              type="email"
              id="email"
              placeholder="email@example.com"
              required
            >
```